docs/react/bootstrap.md

```diff
@@ -50,15 +50,19 @@ my-app
 ├── public
 │   ├── favicon.ico
 │   ├── index.html
-│   └── manifest.json
+│   ├── manifest.json
+│   ├── robots.txt
+│   ├── logo512.png
```

Comment:
I don't think you ran `tree` in a shell, else these items would be sorted alphabetically.

Also `package-lock.json` is missing from the root of `my-app`.